src/app/GitCommands/Settings/AppSettings.cs

```diff
@@ -1678,7 +1678,7 @@ public static ShorteningRecentRepoPathStrategy ShorteningRecentRepoPathStrategy
             set => SetEnum("ShorteningRecentRepoPathStrategy", value);
         }
 
-        public static int MaxPinnedRepositories
+        public static int MaxTopRepositories
         {
             get => GetInt("MaxMostRecentRepositories", 0);
```

Comment:
Add comment why it differ?
(I did not do that when I was asked to rename.)

Reply:
Done (with some little comment refactorings)

Reply:
Thanks for cleaning up